comictaggerlib/main.py

```diff
@@ -177,4 +191,7 @@ def main(self) -> None:
             except Exception:
                 logger.exception("CLI mode failed")
         else:
+            if error and error[1]:
+                print(f"A fatal error occurred please check the log for more information: {error[0]}")  # noqa: T201
+                raise SystemExit(1)
```

Comment:
This is incorrect please remove.
The error is handled in `gui.open_tagger_window(talker_api, self.options, error)`